src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/diff/PrisonerMovementsEventService.kt

```diff
@@ -159,6 +171,13 @@ private fun Prisoner.isSomeOtherMovementOut(previousPrisonerSnapshot: Prisoner?)
   this.inOutStatus == "OUT" &&
     this.status != previousPrisonerSnapshot?.status
 
+private fun isAdmissionAssociatedWithAMerge(offenderBooking: OffenderBooking): Boolean {
+  return offenderBooking.identifiers?.filter { it.type == "MERGED" }?.sortedByDescending { it.whenCreated }?.first()
```

Comment:
very minor but you could have done 
`offenderBooking.identifiers?.filter { it.type == "MERGED" }?.any(it.whenCreated > LocalDateTime.now().minusMinutes(90))`